homeassistant/components/ring/camera.py

```diff
@@ -33,16 +35,12 @@ async def async_setup_entry(
     async_add_entities: AddEntitiesCallback,
 ) -> None:
     """Set up a Ring Door Bell and StickUp Camera."""
-    devices = hass.data[DOMAIN][config_entry.entry_id][RING_DEVICES]
-    devices_coordinator: RingDataCoordinator = hass.data[DOMAIN][config_entry.entry_id][
-        RING_DEVICES_COORDINATOR
-    ]
+    ring_data: RingData = hass.data[DOMAIN][config_entry.entry_id]
+    devices_coordinator = ring_data.devices_coordinator
     ffmpeg_manager = ffmpeg.get_ffmpeg_manager(hass)
 
     cams = []
-    for camera in chain(
-        devices["doorbots"], devices["authorized_doorbots"], devices["stickup_cams"]
-    ):
+    for camera in ring_data.devices.video_devices:
         if not camera.has_subscription:
             continue
```

Comment:
Last one. The whole block here could be one list comprehension. Something like this.
```py
    cams = [
        RingCam(camera, devices_coordinator, ffmpeg_manager)
        for camera in ring_data.devices.video_devices
        if camera.has_subscription
    ]
```

Reply:
That's done